src/components/RichTextEditor/Plugins/ListPlugin/ListPlugin.ts

```diff
@@ -15,15 +17,28 @@ export const createLicStylePlugin = createPluginFactory<ToggleMarkPlugin>({
     key: MARK_TEXT_STYLE,
     isLeaf: true,
 });
+
 export class ListPlugin extends Plugin {
-    constructor(props?: PluginProps) {
+    protected isSoftBreak;
+
+    constructor(props?: ListPluginProps) {
         super(LIST_PLUGIN, {
             leafMarkupElements: [new ListItemContentMarkupElement(), new ListItemMarkupElement()],
             ...props,
         });
+
+        this.isSoftBreak = props?.isSoftBreak ?? false;
     }
 
     plugins() {
-        return [createListPlugin(), createLicStylePlugin()];
+        return [
+            createListPlugin({
+                withOverrides: withList,
+                options: {
+                    isSoftBreak: this.isSoftBreak,
```

Comment:
maybe rename to `useSoftBreak`?

Reply:
Would you have some other name for it, this is more like Hook name :)

Reply:
imho it's unrelated to react, but we can also keep it that way.